### PR TITLE
Add Entity for Booked table.

### PR DESCRIPTION
**Pull request by KovalenkoAndrii**
@TanyaKalistruk @AndriiUngurian

### CODE REVIEW

File: MyBooking.Repositories/BookedRepository.cs [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+                           .Include(i => i.Advert)
+                           .FirstOrDefault(f => f.Id == booked.Id);
+        }
+
+        public bool RemoveById(int bookedId)
+        {
+            Booked booked = GetById(bookedId);
+
+            if (booked == null)
+                throw new Exception("Booked was not found.");
```
Comment (TanyaKalistruk) [inline MyBooking.Repositories/BookedRepository.cs]: Why on 66 line there is ` throw new ArgumentNullException(nameof(booked))`? Why we can't use the same, as there? As for me, this is more understandable

File: MyBooking.Repositories/UserRepository.cs [hunk 2]
```diff
@@ -82,9 +84,10 @@ public bool RemoveById(int userId)
             User user = GetById(userId);
 
             if (user == null)
-            {
                 throw new Exception("User was not found.");
-            }
+
+            if (BookedRepository.GetByUserId(userId) != null)
+                throw new Exception("Landlord of the booked advert cannot be removed.");
```
Comment (TanyaKalistruk) [inline MyBooking.Repositories/UserRepository.cs:90]: Good catch :)

Comment (KovalenkoAndrii) [reply MyBooking.Repositories/UserRepository.cs:90]: ?

File: MyBooking.Repositories/BookedRepository.cs (hunk 1 above)
Comment (KovalenkoAndrii) [reply MyBooking.Repositories/BookedRepository.cs]: Done

**Review by TanyaKalistruk — APPROVED** [review]

**Pull request merged**